### PR TITLE
Cleanup dead French MMRCMS sources and update Lelscan URL

### PR DESCRIPTION
**Pull request by KirinRaikage**
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Review by stevenyomi — CHANGES_REQUESTED** [review]

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mmrcms/SourceData.kt [hunk 1]
```diff
@@ -1,32 +1,19 @@
...
...g,rgb(255,203,112) 0%,rgb(199,81,192) 50%,rgb(65,88,208) 100%);--wp--preset--gradient--pale-ocean: linear-gradient(135deg,rgb(255,245,203) 0%,rgb(182,227,212) 50%,rgb(51,167,181) 100%);--wp--preset--gradient--electric-grass: linear-gradient(135deg,rgb(202,248,128) 0%,rgb(113,206,126) 100%);--wp--preset--gradient--midnight: linear-gradient(135deg,rgb(2,3,129) 0%,rgb(40,116,252) 100%);--wp--preset--duotone--dark-grayscale: url('#wp-duotone-dark-grayscale');--wp--preset--duotone--grayscale: url('#wp-duotone-grayscale');--wp--preset--duotone--purple-yellow: url('#wp-duotone-purple-yellow');--wp--preset--duotone--blue-red: url('#wp-duotone-blue-red');--wp--preset--duotone--midnight: url('#wp-duotone-midnight');--wp--preset--duotone--magenta-yellow: url('#wp-duotone-magenta-yellow');--wp--preset--duotone--purple-green: url('#wp-duotone-purple-green');--wp--preset--duotone--blue-orange: url('#wp-duotone-blue-orange');--wp--preset--font-size--small: 16px;--wp--preset--font-size--medium: 28px;--wp--preset--font-size--large: 32px;--wp--preset--font-size--x-large: 42px;--wp--preset--font-size--larger: 38px;}.has-black-color{color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-g
```
[the remark this answers is not in the document]
Comment (KirinRaikage) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mmrcms/SourceData.kt:14]: I could just repaste whatever was previously there on the file, but I didn't touch it manually anyway.

[the remark this answers is not in the document]
Comment (stevenyomi) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mmrcms/SourceData.kt:14]: You could, but if you have extra time you can try fixing them.

[the remark this answers is not in the document]
Comment (alessandrojean) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mmrcms/SourceData.kt:14]: Some sources have a higher Cloudflare or block access from external IPs, although it is an auto generated file, it's better if you update it manually to only include the specific changes related to the sources you are changing instead.

[the remark this answers is not in the document]
Comment (KirinRaikage) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mmrcms/SourceData.kt]: Thanks for reviewing. I've reverted the changes and updated manually SourceData.kt instead.

**Pull request merged**